<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configurer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../tmp/icons/Солдатик-пограничник.jpg</normaloff>../../../tmp/icons/Солдатик-пограничник.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QDSWidget" name="DSWidget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>320</width>
      <height>256</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>1</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Motion Detector</string>
    </property>
    <widget class="QComboBox" name="DetectorPresetsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="EditDetectorSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ApplyDetectorSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Codec</string>
    </property>
    <widget class="QPushButton" name="editCfg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Config</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>ROI</string>
    </property>
    <widget class="QPushButton" name="downloadROI">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Get ROI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadROI">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Set ROI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearROI">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear ROI</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>401</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="StartVideo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopVideo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>31</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Camera ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CameraID">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="maxVisibleItems">
        <number>4</number>
       </property>
       <property name="maxCount">
        <number>4</number>
       </property>
       <property name="minimumContentsLength">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Thresholds">
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Thresholds</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Marker">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Marker</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="statText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>351</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>571</width>
      <height>25</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>312</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>DSWidget</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>statText</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>connectButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage_Select">
     <property name="title">
      <string>Language Select</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionRussian"/>
    </widget>
    <addaction name="menuLanguage_Select"/>
    <addaction name="actionPort_Config"/>
    <addaction name="actionErase_Config"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="geometry">
     <rect>
      <x>397</x>
      <y>99</y>
      <width>107</width>
      <height>82</height>
     </rect>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <addaction name="actionShowLog"/>
    <addaction name="actionLogSettings"/>
   </widget>
   <widget class="QMenu" name="menuRegisters">
    <property name="title">
     <string>Registers</string>
    </property>
    <addaction name="actionEdit"/>
    <addaction name="actionCamera"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuLog"/>
   <addaction name="menuRegisters"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLanguage_Select">
   <property name="text">
    <string>Language Select</string>
   </property>
  </action>
  <action name="actionErase_Config">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Erase Config</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionLogSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </action>
  <action name="actionCOM1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>COM1</string>
   </property>
  </action>
  <action name="actionCOM2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>COM2</string>
   </property>
  </action>
  <action name="actionCOM3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>COM3</string>
   </property>
  </action>
  <action name="actionCOM4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>COM4</string>
   </property>
  </action>
  <action name="actionPort_Config">
   <property name="text">
    <string>Port Config</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="menuAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDSWidget</class>
   <extends>QWidget</extends>
   <header location="global">qdswidget.h</header>
   <container>1</container>
   <slots>
    <slot>Play()</slot>
    <slot>Stop()</slot>
    <slot>DownloadROI()</slot>
    <slot>UploadROI()</slot>
    <slot>ChangeMarkerMode()</slot>
    <slot>ClearROI()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>StartVideo</tabstop>
  <tabstop>StopVideo</tabstop>
  <tabstop>DetectorPresetsList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>StartVideo</sender>
   <signal>clicked()</signal>
   <receiver>DSWidget</receiver>
   <slot>Play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StopVideo</sender>
   <signal>clicked()</signal>
   <receiver>DSWidget</receiver>
   <slot>Stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadROI</sender>
   <signal>clicked()</signal>
   <receiver>DSWidget</receiver>
   <slot>DownloadROI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadROI</sender>
   <signal>clicked()</signal>
   <receiver>DSWidget</receiver>
   <slot>UploadROI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Marker</sender>
   <signal>clicked()</signal>
   <receiver>DSWidget</receiver>
   <slot>ChangeMarkerMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearROI</sender>
   <signal>clicked()</signal>
   <receiver>DSWidget</receiver>
   <slot>ClearROI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
